tests/probes/yamlfilecontent/test_probes_yamlfilecontent_key.xml

```diff
@@ -41,6 +41,17 @@
       </criteria>
     </definition>
 
+    <definition class="compliance" version="1" id="oval:0:def:4"> <!-- comment="true" -->
+      <metadata>
+        <title></title>
+        <description></description>
+      </metadata>
+      <criteria operator="AND">
+        <criterion comment="multiple_mapping_values" test_ref="oval:0:tst:5"/>
+        <criterion comment="map_target" test_ref="oval:0:tst:6"/>
```

Comment:
I'd prefer to have a separate definition for each of the tests because that makes the debugging easier.

Reply:
Got it, OK

Reply:
Done.